src/components/card.tsx

```diff
@@ -29,9 +30,9 @@ function Card({offer, classType, sizeImage}: CitiesCardProp): JSX.Element {
     <article className={`${cardClasses.wrapper} place-card`}>
       {isPremium && <Badge text={'Premium'} classType={'city'}/>}
```

Comment:
строковые пропсы обычно так передаются 
```suggestion
      {isPremium && <Badge text="Premium" classType="city" />}
```